prow/scripts/cluster-integration/helpers/create-azure-event-hubs-secret.sh

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail  # Fail a pipe if any sub-command fails.
+
+#!/bin/bash
+########################################################################################################################
+#
+# Provision A New Azure EventHub Namespace In The Current Azure Subscription
+#
+# Each Azure EventHubs Namespace can contain a maximum of 10 EventHubs (Knative Channels / Kakfa Topics) which equates
+# to unique combinations of a Event Source / Event Type / Event Version.  Because there is an associated cost with
+# empty or unused EventHub Namespaces, we only want to provision the minimum number required.
+#
+# It is expected that prior to running this script the Azure subscription needs to have sufficient permissions
+# to be able to perform the necessary tasks.  Finally the environment should be setup with "az", "kubectl", "jq",
+# and "base64" on their $PATH.
```

Comment:
base64 is not required as far as I can see

Reply:
Removed